core/src/plugins/filed/python/postgresql/bareos-fd-postgresql.py

```diff
@@ -187,9 +187,10 @@ def __init__(self, plugindef):
         # in end_restore_file(), and this may be mixed up with different files
         self.stat_packets = {}
         self.last_lsn = 0
-        # This will be set to True between select pg_backup_start() and pg_backup_stop().
+        # Those will be set to True between select pg_backup_start() and pg_backup_stop().
         # We backup the cluster files during that time
```

Comment:
```suggestion
        # `backup_running` and `full_backup_running` are set to True between 
        # `pg_backup_start()` and `pg_backup_stop()` (while backing up the cluster files)
```

Reply:
I agree this need to be improved see below